Expand Difficulty Increase mod articles (#12629)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Cover/en.md

*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **Cover** mod hides a part of the playfield similar to [Fade In](/wiki/Gameplay/Game_modifier/Fade_In_(lazer)) and [Hidden](/wiki/Gameplay/Game_modifier/Hidden_(lazer)) but with the possibility of customising its size.

Reviewer: ```suggestion
The **Cover** mod hides a part of the playfield similarly to [Fade In](/wiki/Gameplay/Game_modifier/Fade_In_(lazer)) and [Hidden](/wiki/Gameplay/Game_modifier/Hidden_(lazer)), but with the possibility of customising the size of obscured area.
```

## Customisation

![Cover customisation options in the game client](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/CO.jpg)

> - **Coverage** (0,2-0,8, default 0,5): The proportion of playfield height that notes will be hidden for.

Reviewer: ```suggestion
- **Coverage** (0,2-0,8, default: 0,5): The proportion of playfield height for which notes will be hidden.
```

> - **Direction** (default: Along scroll): The direction on which the cover is applied.
>   - **Along scroll**: Cover the playfield from above.
>   - **Against scroll**: Cover the notes from below.

Reviewer: consistency
```suggestion
- **Direction** (default: Along scroll): The direction on which the cover is applied.
  - **Along scroll**: Cover the playfield from above.
  - **Against scroll**: Cover the playfield from below.
```
